Sources/OTelCore/OTel+Configuration+EnvironmentOverrides.swift

```diff
@@ -123,6 +124,51 @@ extension OTel.Configuration.LogsConfiguration {
     }
 }
 
+extension OTel.Configuration.TracesConfiguration.SamplerConfiguration {
+    internal mutating func applyEnvironmentOverrides(environment: [String: String]) {
+        if let sampler = environment.getStringValue(.sampler) {
+            switch sampler {
+            case "always_on": self = .alwaysOn
+            case "always_off": self = .alwaysOff
+            case "traceidratio": backing = .traceIDRatio
+            case "parentbased_always_on": self = .parentBasedAlwaysOn
+            case "parentbased_always_off": self = .parentBasedAlwaysOff
+            case "parentbased_traceidratio": backing = .parentBasedTraceIDRatio
+            case "parentbased_jaeger_remote": self = .parentBasedJaegerRemote
+            case "jaeger_remote": self = .jaegerRemote
+            case "xray": self = .xray
+            default: self = .parentBasedAlwaysOn
+            }
+        }
+        if let samplerArgument = environment.getStringValue(.samplerArgument) {
+            switch backing {
+            case .traceIDRatio, .parentBasedTraceIDRatio:
+                guard let samplingProbability = Double(samplerArgument) else { break }
+                argument = .traceIDRatio(samplingProbability: samplingProbability)
+            case .jaegerRemote, .parentBasedJaegerRemote:
+                // Example: endpoint=http://localhost:14250,pollingIntervalMs=5000,initialSamplingRate=0.25
```

Comment:
Is this implementation necessary given we may very well not implement the Jaeger samplers ever? I understand that it's spec-compliant to implement it but if we're fatal-erroring based on the sampler not being implemented anyways I don't see the advantage of including it here.

Reply:
Yeah, you're probably right. This was me implementing these bits in parallel. Once we do the "handle the unsupported options bit" we might as well rip this out.